tools/AzureRM.BootStrapper/AzureRM.BootStrapper.Tests.ps1

```diff
@@ -0,0 +1,231 @@
+Import-Module -Name AzureRM.Bootstrapper
```

Comment:
What are the requirements for runnjing these tests?  Looks like we have some external depednencies that I don't see referenced anywhere